### PR TITLE
Transfer List Dark Mode - RSC-1055

### PR DESCRIPTION
**Pull request by rpokorny**
https://issues.sonatype.org/browse/RSC-1055

### CODE REVIEW

File: lib/src/base-styles/_nx-colors.scss [hunk 1]
```diff
@@ -368,6 +380,15 @@
     --nx-color-checkbox-disabled-background: var(--nx-color-disabled);
     --nx-color-checkbox-disabled-inner-border: var(--nx-color-disabled);
     --nx-color-checkbox-disabled-icon: var(--nx-color-component-background);
+
+    // NxTransferList
+    --nx-color-transfer-list-item-background: var(--nx-swatch-indigo-20);
+    --nx-color-transfer-list-item-background-hover: var(--nx-swatch-indigo-15);
```
Comment (cindyko226) [inline lib/src/base-styles/_nx-colors.scss:386]: In the sketch, the color for background hover is set to `nx-indigo-10`

File: lib/src/base-styles/_nx-colors.scss [hunk 2]
```diff
@@ -368,6 +380,15 @@
...
+    --nx-color-transfer-list-item-background-active: var(--nx-swatch-indigo-35);
```
Comment (cindyko226) [inline lib/src/base-styles/_nx-colors.scss:387]: In the sketch, the color for background active is set to `nx-indigo-30`

**Comment by cindyko226** [comment]
Missing variables documentation

File: lib/src/components/NxTransferListHalf/NxTransferListHalf.scss [hunk 3]
```diff
@@ -115,28 +116,28 @@ $-visible-item-count: 10;
...
     &:hover {
-      background-color: var(--nx-color-interactive-background-hover);
-      border-color: var(--nx-color-interactive-border-hover);
+      background-color: var(--nx-color-transfer-list-item-background-hover);
+      border-color: var(--nx-color-transfer-list-item-border-hover);
     }
-  
+
     &:focus-within:not(:active) {
       outline: 2px solid var(--nx-color-interactive-border-focus);
```
Comment (EmilyBermudez88) [inline lib/src/components/NxTransferListHalf/NxTransferListHalf.scss:124]:
```suggestion
      outline: var(--focus-outline-width) solid var(--nx-color-interactive-border-focus);
```

File: lib/src/base-styles/_nx-colors.scss (hunk 2 above)
Comment (rpokorny) [reply lib/src/base-styles/_nx-colors.scss]: > In the sketch, the color for background active is set to nx-indigo-30

465f00d2e

**Comment by rpokorny** [comment]
> The focus color of the "Move Up/Down" arrows is currently inheriting the color from .nx-btn--icon-only instead of using purple-80

I bet that's a mockup bug.  At this point those buttons are identical to regular icon-only buttons in every way other than this.  @JasonDSwearingen should the focus border color be made consistent as well?

**Comment by rpokorny** [comment]
> Missing variables documentation

[a72fe42](https://github.com/sonatype/sonatype-react-shared-components/pull/1068/commits/a72fe42cae4dfe624b9a4c9daf32c8c76a2c2ad0)

**Comment by rpokorny** [comment]
> Just noticed there is comment in the ticket https://issues.sonatype.org/browse/RSC-1055?focusedId=1259771&page=com.atlassian.jira.plugin.system.issuetabpanels:comment-tabpanel#comment-1259771 to update the snapshots of nxFieldset that include the transfer list.

Thanks for catching that.  f235d632a

**Pull request merged**